test/test_output_engine.py

```diff
@@ -472,21 +473,28 @@ def test_output_csv(self):
 
     def test_output_pdf(self):
         """Test formatting output as PDF"""
-        output_pdf(self.MOCK_PDF_OUTPUT, False, 1, "cve_test.pdf", False)
-        with open("cve_test.pdf", "rb") as f:
-            pdf = pdftotext.PDF(f)
-            # Only interested in section 3 of the report which contains table of CVEs. This is on the second page
-            page = pdf[1]
-            # Find start of section 3 header
-            section2_start = page.find("3. List of Identified Vulnerabilities")
-        self.assertEqual(
-            page[section2_start:]
-            .replace(" ", "")
-            .replace("\r", "")
-            .replace("\n", "")
-            .strip(),
-            self.PDF_OUTPUT.replace(" ", ""),
-        )
+        if (
+            importlib.util.find_spec("reportlab") is not None
+            and importlib.util.find_spec("pdftotext") is not None
+        ):
```

Comment:
isn't it better to skip the test when reportlab isn't installed using a [skip decorator](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) instead of passing the test when it's not installed?

Reply:
@BreadGenie I agree. Can I simply pass the expression as is to the skip decorator?

Reply:
yep, you can use either `skipIf(condition, reason)` or `skipUnless(condition, reason)`